srcs/app/http/HttpRequestHandler.cpp

```diff
@@ -36,75 +68,206 @@ void		HttpRequestHandler::HandleRequest_() {
 		return;
 	}
 	SetKeepAlive_(*request);
-	if (request->GetMethod() == "GET")
+	request_location_ = new RequestLocation(server_config_, request->GetPath());
+	if (request->GetMethod() == "GET") {
```

Comment:
I would make this request handling with something like an interface, abstract class or a design patter such as factory (would have to see if it would match this implementation) I think this way it is a little bit coupled... imagine you have to add more HTTP methods, you will have to expand more and more the if else chain.

I think for a first approach your solution is really good though and does no make sense to modify it since it is out of the scope of this task. Let me add a new issue and work on it myself!

Reply:
Sure, go for it!

I want to point out something about this` if/else if/else`. Currently, the `else` part never executes because in `HttpRequest` class, only `GET`, `POST`, and `DELETE` methods are valid.